setup.py

```diff
@@ -17,7 +17,7 @@
 
 setup(
     name='python-dependency-verifier',
-    version='0.1',
+    version='0.2',
```

Comment:
python versions are usually formatted as follows:
X.Y.Z

where X is a major release
Y is a substantial change
Z is a minor change

while it's not a must, I would use this format. The reason why we don't use it in Cloudify is because we currently don't release minor changes.